.bazelrc

```diff
@@ -106,6 +106,21 @@ build:windows --enable_runfiles
 # used by some KJ macros, and understands the `.c++` extension by default.
 build:windows --compiler=clang-cl
 
+# TODO: Only enable this for what would otherwise be fastbuild
+# The Windows fastbuild bazel configuration is broken in that it necessarily generates PDB debug
+# information while the Linux and macOS toolchains only compile with debug information in the dbg
+# configuration or when requested with the -g flag. This causes huge increases in compile time and
+# disk/cache space usage – a single test may come with a 490MB PDB file.
+# Use the opt configuration and disable optimizations as a workaround.
+
+build:windows -c opt
+build:windows --copt='-O0' --host_copt='-O0'
+build:windows --copt='/Od' --host_copt='/Od'
+build:windows --copt='-g0' --host_copt='-g0'
```

Comment:
Do `-g0`and `-O0` now apply to all `compilation_mode` variants on Windows?

Reply:
There was a TODO to only apply this to what would usually be fastbuild – I changed it to only apply if `--config=windows_no_dbg` is enabled, the CI test runner does that now.

Reply:
Super, thanks!